### ISSUE TITLE
title grabbing error

### ISSUE THREAD

**Issue by zwk5131** [Ruoyu-Klaus/obsidian-clipper#4]
Hey, this is an amazing program :D great work !
Could you fix the naming of the note... it can't save the hole name, when naming the file. it cuts it off after a few characters...

**Comment by Ruoyu-Klaus** [comment]
Hi, Thank you for the feedback. 
so far, If a file name longer than 16 characters is detected, it will cut the rest out to ensure OS file system works as usual.
I'll consider to edit it not more than 50 characters or make it configurable in setting page.

**Comment by zwk5131** [comment]
Yes that would be amazing ! :D I think 60 would be enough and after that maybe cutting on a white-space :)

**Comment by alexzadeh** [comment]
+1 for this! Love your extension. It's the only real flaw, but I've still posted a 5* review.

Small request - It would be cool to have the ability to customise the header info above the content. I don't need the title duplicated, but I like having the link and date.

**Comment by zwk5131** [comment]
Another idea for grabbing title would be to grab the tabname from the website + png...
